README.md

```diff
@@ -157,6 +157,6 @@ Environmental variables:
 | `max_inflight`              | Yes          | Limit the maximum number of requests in flight |
 | `mode`                      | Yes          | The mode which of-watchdog operates in, Default `streaming` [see doc](#3-streaming-fork-modestreaming---default). Options are [http](#1-http-modehttp), [serialising fork](#2-serializing-fork-modeserializing), [streaming fork](#3-streaming-fork-modestreaming---default), [static](#4-static-modestatic) |
 | `prefix_logs`             | Yes          | When set to `true` the watchdog will add a prefix of "Date Time" + "stderr/stdout" to every line read from the function process. Default `true` |
-
+| `log_buffer_size` | The amount of bytes to read from stderr/stdout for log lines. The efault is bufio.MaxScanTokenSize |
```

Comment:
```suggestion
| `log_buffer_size` | The amount of bytes to read from stderr/stdout for log lines. The default is bufio.MaxScanTokenSize |
```

Reply:
I also think it would be good to add some more docs here about what happens when this limit is hit? is the line truncated or an error is returned? what is the structure of the error so that people know what is happening and debug it accordingly? 

Your PR description mentions an error but never actually gives a clear example of what that error looks like. It would be nice to have this information n the PR description and the README so that it is more discoverable

Reply:
I would like to have it too, but I'm still waiting on the user to raise the issue.

This PR is preemptive.

Reply:
This has been resolved. I've referenced #99 in the PR description so that someone can clearly find the paper trail.